Tests/winswTests/Util/CLITestHelper.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.IO;
+using JetBrains.Annotations;
+using winsw;
+
+namespace winswTests.Util
+{
+    /// <summary>
+    /// Helper for WinSW CLI testing
+    /// </summary>
+    public static class CLITestHelper
```

Comment:
I would avoid making major project structure changes in winsw-1.x
For such purpose I would propose to conduct such refactoring in https://github.com/kohsuke/winsw/tree/winsw-2.0 (it's mapped to https://github.com/kohsuke/winsw/pull/71)